### ISSUE TITLE
[Svelte TypeScript] Property 'builtIn' is missing in type with Custom Components

### ISSUE THREAD

**Issue by D-Marc1** [BuilderIO/builder#1728]
The following code shown consists of the pertinent parts. Other parts of the code were omitted for brevity.

**Code**

```svelte
<script lang="ts">
const customComponents = [
  {
    component: Counter,
    name: 'Counter',
    image: 'https://cdn-icons-png.flaticon.com/512/6134/6134688.png',
    inputs: [
      {
        name: 'count',
        type: 'number',
        defaultValue: 0
      },
    ],
  },
]
</script>

<RenderContent
  model="page"
  content={data.content}
  apiKey={BUILDER_PUBLIC_API_KEY}
  customComponents={customComponents}
/>
```

**The full message is:**

> Type '{ component: typeof Counter__SvelteComponent_; name: string; image: string; inputs: { name: string; type: string; defaultValue: number; }[]; }[]' is not assignable to type 'RegisteredComponent[]'.
  Type '{ component: typeof Counter; name: string; image: string; inputs: { name: string; type: string; defaultValue: number; }[]; }' is not assignable to type 'RegisteredComponent'.
    Property 'builtIn' is missing in type '{ component: typeof Counter; name: string; image: string; inputs: { name: string; type: string; defaultValue: number; }[]; }' but required in type 'ComponentInfo'.

**Comment by mrkoreye** [comment]
@samijaber can you take a look?

**Comment by samijaber** [comment]
@D-Marc1 Fix is live on `1.0.6`
